### PR TITLE
DUB: Require most recent compiler / dub, simplify

### PR DESCRIPTION
**Pull request by Geod24**
Note that LDC `v1.27.0-beta1` has been released so it's not going to stay the most recent version for long.

### CODE REVIEW

**Comment by Geod24** [comment]
This allowed me to find #2158 BTW, so an immediate win.

**Comment by Geod24** [comment]
Alpine needs to be updated

**Comment by Geod24** [comment]
https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/22088

**Comment by Geod24** [comment]
Actually what we needed was https://github.com/bosagora/docker-agora-builder/pull/13 since we don't use upstream images (but it doesn't hurt to update them either).

**Comment by Geod24** [comment]
@omerfirmak : I blame you: https://github.com/dlang/dub/issues/2142

**Comment by omerfirmak** [comment]
> @omerfirmak : I blame you: [dlang/dub#2142](https://github.com/dlang/dub/issues/2142)

That is happening a lot these days.

File: .dockerignore [hunk 1]
```diff
@@ -10,7 +10,8 @@ README.md
 .dub/
 build/
 docs/
-source/scpp/build/
+source/scpp/build/*.o
+source/scpp/build/*.obj
```
Comment (Geod24) [inline .dockerignore:14]: Also a workaround for the DUB issue.

**Comment by Geod24** [comment]
💚

**Review by linked0 — APPROVED** [review]
LGTM

**Review by kchulj — APPROVED** [review]

**Pull request merged**